Lib/ctypes/wintypes.py

```diff
@@ -82,8 +88,10 @@ def __repr__(self):
 HMONITOR = HANDLE
 HPALETTE = HANDLE
 HPEN = HANDLE
+HRESULT = LONG
 HRGN = HANDLE
 HRSRC = HANDLE
+HSZ = HANDLE
```

Comment:
This one should come after `HSTR` to be alphabetically sorted.

Reply:
Okay, I'll make the changes now